frontend/public/components/monitoring/metrics.tsx

```diff
@@ -438,12 +453,21 @@ export const QueryBrowserPage = withFallback(() => {
       </div>
       <div className="row">
         <div className="col-xs-12">
-          <QueryBrowser
-            defaultTimespan={30 * 60 * 1000}
-            disabledSeries={_.map(queries, 'disabledSeries')}
-            onDataUpdate={onDataUpdate}
-            queries={_.map(queries, 'query')}
-          />
+          {queryStrings.join('') === ''
+            ? <div className="query-browser__wrapper graph-empty-state">
+              <EmptyState variant={EmptyStateVariant.full}>
+                <EmptyStateIcon size="sm" icon={ChartLineIcon} />
+                <Title size="sm">No Query Entered</Title>
+                <EmptyStateBody>Enter a query in the box below to explore metrics for this cluster</EmptyStateBody>
```

Comment:
nit: we usually include a period if it's a complete sentence

```suggestion
                <EmptyStateBody>Enter a query in the box below to explore metrics for this cluster.</EmptyStateBody>
```

Reply:
Addressed by https://github.com/openshift/console/pull/2096